README.md

```diff
@@ -34,8 +34,7 @@
   - [Docker](#docker)
   - [Install from sources](#install-from-sources)
 - [Usage](#usage)
-  - [`ipfs-cluster-service`](#ipfs-cluster-service)
-  - [`ipfs-cluster-ctl`](#ipfs-cluster-ctl)
+  - [`Quickstart`](#quickstart)
```

Comment:
You did something weird if this is in the diff. You probably need something like:

`git remote add upstream https://github.com/ipfs/ipfs-cluster.git`
`git fetch upstream`
`git rebase upstream/master`

and fix conflicts along the way:

`fix conflict`
`git add <problem file>`
`git rebase --continue`

at the end

`git push origin <your branch> --force`

so that only your changes are in this PR.